Gemfile

```diff
@@ -25,9 +25,7 @@ source "http://rubygems.org"
   gem 'oj', '~> 2.12.14'
 
   gem "capistrano",           "~> 2.14.1" #      :require => "capistrano"
-  # gem 'capistrano-autoscaling', "0.0.8.3", :path => "../capistrano-autoscaling/"
-  gem 'capistrano-autoscaling', "0.0.8.5", :git => "git://github.com/concord-consortium/capistrano-autoscaling",  :branch => "concord"
-  # gem 'capistrano-autoscaling', "~> 0.0.9git",  :git => "git://github.com/yyuu/capistrano-autoscaling.git",  :branch => "develop"
+  # gem 'capistrano-autoscaling', "0.0.8.5", :git => "git://github.com/concord-consortium/capistrano-autoscaling",  :branch => "concord"
```

Comment:
@scytacki, I'm assuming it's not necessary anymore. It's commented out and not removed, as there are still some scripts referencing it. I didn't clean them up, as it seems like a separate task and I'm not sure, maybe we want to keep them in case we ever go back to old deployment method (feels unlikely). 😉

Reply:
👍